process/txcost/converters_test.go

```diff
@@ -0,0 +1,20 @@
+package txcost
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRemoveLatestArgumentFromDataField(t *testing.T) {
+	t.Parallel()
+
+	dataField := removeLatestArgumentFromDataField("function@arg1@arg2@arg3@shouldBeRemoved")
```

Comment:
please add 2 tests here:
- one with empty string as parameter
- one with only one string parameter that does not contain `@`

Reply:
done and found o problem. :+1: